src/main/twirl/views/sectionPreview.scala.html

```diff
@@ -13,7 +13,7 @@
     f"RIFS$id%05d"
 }
 
-@main(s"Event title", backLink=Some(BackLink("Return to application overview",controllers.routes.ApplicationController.show(appOverview.id).url))) {
+@main(formSection.title, backLink=Some(BackLink("Return to application overview",controllers.routes.ApplicationController.show(appOverview.id).url))) {
```

Comment:
It's the section title that appears in the page body - Event Title, Topics and speakers etc.  Essentially it is the text which appears in the browser title bar/ tab etc & was hardcoded to be "Event Title".  this parameterises it to be the same as the section title (as per sectionForm.scala.html).